proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
```

Comment:
Delete this Comparisons.adoc file.  This is the subject of PR #74 and still needs to be fleshed out before posting.